src/guides/v2.3/extension-dev-guide/view-models.md

```diff
@@ -14,6 +14,19 @@ Use this approach anytime you need to inject functionality into template files a
  {:.bs-callout-info}
 View models are available in Magento 2.2 onwards. If your code must be compatible with older versions of Magento, consider adding your logic to blocks. For more information about backward compatibility, see [Backward compatibility]({{ page.baseurl }}/contributor-guide/backward-compatible-development/).
 
+The following is an example of helper class directly used in `Magento/Catalog/view/frontend/templates/product/widget/new/content/new_grid.phtml` templates.
+
+```xml
+$postDataHelper = $this->helper(Magento\Framework\Data\Helper\PostHelper::class);
```

Comment:
Hi @ajithkumar-maragathavel. The described example is good but I'm not sure we should show how to use helpers in view models documentation. If there's a way to use the view model in the similar way, we may put this information. Otherwise, using helper is out of context.

Reply:
@rogyar I will provide another example below the current one to show how we can achieve the same functionality using the view model concept. Will that works?
Because my whole idea is to inform developers to use the view models instead of the helper class. Since It is the best practice.

If it is not okay, Please provide me some suggestions.
Thanks for the review.

Reply:
Hi @ajithkumar-maragathavel. I got your idea, thank you for the explanation. My point is, I'm not sure we should show the helper usage in the example. If we show an example of something, the reader get's it as a good practice first of all (even if then we have a mention that it's not a good practice). 
If you have a possibility to replace the helper example with the view model example somehow, it would make this topic much more useful. 

Thank you!

Reply:
@rogyar Thanks for your suggestion. I can understand your point now. I will come up with the view model example.
Thanks

Reply:
@rogyar I have updated the document with the view model example from Magento_Catalog module. Please review it.
Let me know your thought on this example.
Thanks.